app/controllers/chapters_controller.rb

```diff
@@ -19,9 +20,9 @@ class ChaptersController < ApplicationController
   def chapter_1
     @chapter_number = 1
     @chapter_last_updated = 'May 2019'
-    @next_chapter_title = YAML.load(File.open("#{Rails.root}/lib/data/content/chapter-2.yml", 'r'))['menu_title']
+    @next_chapter_title = load_yaml("lib/data/content/chapter-2.yml")['menu_title']
```

Comment:
I was thinking to extract the base path somewhere so that can be shared withouth specifying it everytime.
Perhaps a private method within the ApplicationController might help:

```
def content_base_path
  'lib/data/content/'
end
```

Reply:
Yep, I agree. Will implement the suggestions above too :)